rasa/core/test.py

```diff
@@ -679,16 +732,20 @@ async def _collect_story_predictions(
     logger.info(f"Evaluating {number_of_stories} stories\nProgress:")
 
     action_list = []
+    entity_results = []
```

Comment:
It's only entities predicted by policies, right?
```suggestion
    policy_entity_results = []
```